app/controllers/users_controller.rb

```diff
@@ -4,8 +4,11 @@ def index
   end
 
   def show
-    @user = 'users'
-    # @posts = @user.posts
-    @posts = 'posts'
+    @user = User.find_by(id: params['id'])
+    @posts = Post.all || 'No posts yet'
+    @post = []
+    @posts.each do |post|
+      @post << post if post.author_id == @user.id
+    end
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/Reggeditt/blog-rails/assets/77069011/2c1dca57-3389-478c-a502-435593b2d0e8)

Please use the `#recent_posts` method you created in your `User` model for this view. 🤓 🔧

Reply:
fixed